generate-stackbrew-library.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 declare -A aliases
 aliases=(
-	[0.12.0]='0 latest'
+	[0.12.2]='0 latest'
```

Comment:
`0` and `latest` for the raspberry pi 1 should point to `0.10.38` until the compilation issue is resolved